src/components/notes/offline-indicator.styl

```diff
@@ -0,0 +1,26 @@
+.grid
+    padding 1rem
+    font-size 1rem
+    .banner &
+        font-size 0.875rem
+        line-height 1.3
+        padding 0.25rem 1rem 0.25rem 1rem
+        margin 0
+        width 100%
+
+.chip
+    background-color var(--silver)
+    opacity 60%
+
+.tooltip
+    background-color var(--paleGrey) !important
+    border-radius 8px !important
```

Comment:
Why those important? When I take a look at https://v3.material-ui.com/demos/tooltips/ I can see a `lightTooltip` whiteout any important

Reply:
Their CSS is loaded after ours. I could also use an higher specificity for the selector but I  it doesn't seem really better.

Reply:
Ça me parait difficilement imaginable qu'on ne puisse pas surcharger ce qu'on veuille via leur API CSS ou via les thèmes et qu'on doive passer par des important. On target bien les bons éléments ? Genre on est sur devoir toucher au .tooltip et non au  .popper ?

Reply:
je n'ai pas réussi à changer l'ordre d'insertion des CSS de MUI. Je suis passé par le CSS-in-JS du coup